src/test/resources/data/StudentFeedbackQuestionSubmitPageUiTest.json

```diff
@@ -29,6 +29,7 @@
             "name":"Teammates Test",
             "email":"SFQSubmitUiT.instr@gmail.tmt",
             "role":"Co-owner",
+            "isDisplayedToStudents":true,
```

Comment:
The test starting failing because of your modifications right? In other words, StudentFeedbackQuestionSubmitPageAction was also incorrectly showing hidden instructors?